### PR TITLE
Load list of tags to skip from file

### PR DESCRIPTION
**Pull request by msherman64**
kolla-ansible supports the --skip-tags option

This adds a simple list of tags to a file. Uncomment lines to skip.

### CODE REVIEW

File: kolla-skip-tags [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
```
Comment (diurnalist) [inline kolla-skip-tags]: Doesn't need this does it?

File: cc-ansible [hunk 2]
```diff
@@ -325,4 +325,11 @@ if [[ "${POSARGS[*]}" =~ post-deploy ]]; then
 fi
 kolla_args+=("${POSARGS[@]}")
 
+#load tags to skip
+ANSIBLE_SKIP_TAGS="$(grep -v "^#" ./kolla-skip-tags | paste -sd ',')"
+if [[ "${ANSIBLE_SKIP_TAGS}" ]]; then
```
Comment (diurnalist) [inline cc-ansible]: Huh, I didn't know it could be that simple. Is there a difference b/w this and `-n` ?

File: cc-ansible [hunk 3]
```diff
@@ -325,4 +325,11 @@ if [[ "${POSARGS[*]}" =~ post-deploy ]]; then
...
+  kolla_args+=("--skip-tags")
+  kolla_args+=("${ANSIBLE_SKIP_TAGS[@]}")
```
Comment (diurnalist) [inline cc-ansible]: Do you need the `[@]`?

**Review by diurnalist — APPROVED** [review]
Looks good, I like that they are in order!

File: kolla-skip-tags (hunk 1 above)
Comment (msherman64) [reply kolla-skip-tags]: This was just to enable syntax highlighting in general. It's getting evaluated by bash / gnu functions, so it seemed reasonable to include.

**Comment by msherman64** [comment]
The list of tags was generated by running the raw ansible-playbook command with `--list-tags`, then manually parsing it.
It might be good to have some kind of CI process generate that list, so it stays up to date.

File: cc-ansible (hunk 3 above)
Comment (msherman64) [reply cc-ansible]: I need to check, that's left over from when it was getting interpreted as an array.

File: cc-ansible (hunk 2 above)
Comment (msherman64) [reply cc-ansible]: With no arguments, `[[ expr ]]` tests for "is null", where `-n` tests for "is length nonzero"
From what I can find: https://unix.stackexchange.com/a/280432 they result in the same behavior, but `-n` is semantically clearer.

**Pull request merged**